src/app/components/Search/ResultsList.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {
+  isEmpty as _isEmpty,
+} from 'underscore';
+import Instances from './Instances';
+
+export const ResultsList = (results) => {
+  if (_isEmpty(results.results)) {
+    return null;
+  }
+
+  return (
+    <div>
+      <h2>Works</h2>
+      <ul className="nypl-results-list">
+        {
+          results.results.map((result, i) => (
+            <li className="nypl-results-item" key={i.toString()}>
+              <h3>
+                <a href={`/work/detail/${result._id}`}>
+                  {result['_source'].title} &ndash; {result['_source'].entities[0].name}
```

Comment:
Just leaving a note that as we work on the API the response block will become easier to parse/more logical, so we won't have to deal with the nested `data` structure and the underscore variable names.

Reply:
A modeling function can be created to create the data structure _for_ the front-end. Whenever the data structure in the backend gets updated, you'd only have to update that one modeling function to update the data structure for the whole app.
Perhaps something along the lines of  https://github.com/NYPL-discovery/discovery-front-end/blob/development/src/app/utils/item.js#L45